src/overlays/actors/ovl_Bg_Zg/z_bg_zg.c

```diff
@@ -1,16 +1,32 @@
+/*
+ * File: z_bg_zg.c
+ * Overlay: ovl_Bg_Zg
+ * Description: Metal bars (Ganon's Castle)
+ */
+
 #include "z_bg_zg.h"
 
 #define ROOM 0x00
 #define FLAGS 0x00000010
 
-void BgZg_Init(BgZg* this, GlobalContext* globalCtx);
-void BgZg_Destroy(BgZg* this, GlobalContext* globalCtx);
-void BgZg_Update(BgZg* this, GlobalContext* globalCtx);
-void BgZg_Draw(BgZg* this, GlobalContext* globalCtx);
+static void BgZg_Init(BgZg* this, GlobalContext* globalCtx);
+static void BgZg_Destroy(BgZg* this, GlobalContext* globalCtx);
+static void BgZg_Update(BgZg* this, GlobalContext* globalCtx);
+static void BgZg_Draw(BgZg* this, GlobalContext* globalCtx);
+static void func_808C0C50(BgZg* this);
+static s32 func_808C0C98(BgZg* this, GlobalContext* globalCtx);
+static s32 func_808C0CC8(BgZg* this);
+static void func_808C0CD4(BgZg* this, GlobalContext* globalCtx);
+static void func_808C0D08(BgZg* this, GlobalContext* globalCtx);
+static void func_808C0EEC(BgZg* this, GlobalContext* globalCtx);
 
-/*
-const ActorInit Bg_Zg_InitVars =
-{
+static const ActorFunc funcTbl[] = { (ActorFunc)func_808C0CD4, (ActorFunc)func_808C0D08 };
+
+static InitChainEntry initChain[] = { ICHAIN_VEC3F_DIV1000(scale, 1000, ICHAIN_STOP) };
```

Comment:
Always have trailing commas in init chain arrays so clang-format handles it properly. And I would have trailing commas in the function tables as well.